docs/getting_started/scaleway.md

```diff
@@ -111,3 +107,69 @@ kops update cluster mycluster.k8s.local --yes
 ```bash
 kops delete cluster mycluster.k8s.local --yes
 ```
+
+# Terraform support
+
+kOps offers the possibility to generate a Terraform configuration corresponding to the cluster that would have been created directly otherwise.
+
+You can find more information on the dedicated page on [kOps Terraform support](../terraform.md) or [Scaleway's Terraform provider's documentation](https://github.com/scaleway/terraform-provider-scaleway).
+
+## For clusters without load-balancers
+
+This concerns clusters using Scaleway DNS. For this type of clusters, things are pretty simple.
+
+```bash
+kops create cluster --cloud=scaleway --name=mycluster.mydomain.com --zones=fr-par-1 --target=terraform --out=$OUTPUT_DIR
+cd $OUTPUT_DIR
+terraform init
+terraform apply
+```
+kOps will generate a `kubernetes.tf` file in the output directory of your choice, you just have to initialize Terraform and apply the configuration.
```

Comment:
Can this name be changed using an argument?

Reply:
I don't think so, the name of the file is defined [here](https://github.com/kubernetes/kops/blob/9816fd941624f73e5b2a819bce58f443d703042e/upup/pkg/fi/cloudup/terraform/target_hcl2.go#L57)